src/part_1/chapter_2/chapter_1-2.tex

```diff
@@ -137,7 +137,7 @@ \subsection{Subspaces and bases}
 
 \subsection{Affine subspaces}
 
-Affine subspaces are not subspaces, because they do not contain the origin (recall that the definition of subspaces allow for $a$ and $b$ to be zero). Nevertheless, $S$ has the \emph{same dimension} as $S_0$.
+Affine subspaces are not subspaces, because they do not contain the origin (recall that the definition of subspaces allows for $a$ and $b$ to be zero). Nevertheless, $S$ has the \emph{same dimension} as $S_0$.
 
 Affine subspaces give a framework for representing linear programming problems algebraically.  Specifically, let $A$ be a $m \times n$ matrix with $m < n$ (which will always be the case in linear programming models in this form as we will see) and $b$ a $m$-dimensional vector. Then, let 
```

Comment:
m<n always being the case feels incorrect to me.

Reply:
It is because of an underlying assumption that we add slack variables for inequalities and artificial variables for equalities. So a problem with 1 variable and 10 constraints will end up with at least 11 variables. Perhaps we could simply remove it at this point to avoid confusion.